app/utils/config.ts

```diff
@@ -98,33 +98,23 @@ export const useDocsConfig = ({
   const match = matches[matches.length - 1]
   const params = useParams()
   const version = params.version!
-  const framework = localStorage.getItem('framework') || 'react'
+  const localStorageFramework = localStorage.getItem('framework')
+  const framework = localStorageFramework
```

Comment:
Fixes missing docs when the framework saved in localStorage isn't available for that project